README.md

```diff
@@ -164,6 +164,14 @@ Replace [`static/favicon.ico`](//github.com/escalate/hugo-split-theme/tree/maste
 favicon = "favicon.ico"
 ```
 
+### Add custom CSS
+Add the following to `config.toml`:
+```toml
+[params]
+  customCSS = "/css/style.css"
+```
+
+Now add your custom CSS to the defined location (here: `/static/css/style.css`)
```

Comment:
Could you please use the path like the theme does?

/static/css/style.css -> /static/assets/css/style.css

This is more convenient.